### PR TITLE
IBM-109: unauthorized

### PR DESCRIPTION
**Pull request by cksanders**
ES6 using the babel 6 cli

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,63 +4,27 @@
...
    };
    
-   var client = new XMLHttpRequest();
-   
-   keycloakRedirect.authenticate(config, client, window);
-```
-
-the `backend`, `clientId` and `keycloakUrl` keys on the config object are mandatory.
-
-## Run the build scripts
```
Comment (chrisns) [inline README.md:36]: why have you removed all this documentation?

File: package.json [hunk 2]
```diff
@@ -1,11 +1,13 @@
 {
   "name": "keycloak-redirect",
-  "version": "1.0.1",
+  "version": "1.1.7",
   "description": "keycloak redirect browserify",
   "main": "index.js",
```
Comment (chrisns) [inline package.json]: but you've deleted this file?

File: README.md (hunk 1 above)
Comment (cksanders) [inline README.md:36]: I have updated the readme; to be more helpful and removed redundant code / features etc.

You are probably better off reading the full file: https://github.com/UKHomeOffice/keycloakjs-redirect/pull/12/commits/b49924b5cf4d4d35d44a0bd2a055fce77ca601a4

File: package.json [hunk 3]
```diff
@@ -1,11 +1,13 @@
 {
   "name": "keycloak-redirect",
-  "version": "1.0.1",
+  "version": "1.1.7",
   "description": "keycloak redirect browserify",
-  "main": "index.js",
+  "main": "dist/",
```
Comment (chrisns) [inline package.json]: this should be a file not a directory (iirc)

File: README.md (hunk 1 above)
Comment (chrisns) [inline README.md:36]: sure, but the `npm test` is probably a still valid line?
along with something about the new `npm compile`

Comment (cksanders) [inline README.md:36]: may be, however I have added a precommit task to run tests

**Pull request merged**